otherlibs/systhreads/st_stubs.c

```diff
@@ -978,40 +978,63 @@ static st_retcode caml_threadstatus_wait (value wrapper)
   CAMLreturnT(st_retcode, retcode);
 }
 
+#define caml_set_current_thread_name_warning(w)                                \
+  do {                                                                         \
+    if (caml_runtime_warnings_active()) {                                      \
+      fprintf(stderr, "Error while setting thread name: %s\n", w);             \
```

Comment:
Other runtime warnings seem to use an `[ocaml] ` prefix to attempt to clarify the source of the message -- see other occurrences of `caml_runtime_warnings_active`.

Reply:
That's a good idea! Just added it!